src/core_seaice/Registry.xml

```diff
@@ -1828,17 +1828,27 @@
 			<var name="mobileFraction"/>
 			<var name="newlyFormedIce"/>
 			<var name="maximumIcePresence"/>
-			<var name="bergAreaCategory"/>
-			<var name="bergMassCategory"/>
-			<var name="bergLength"/>
-			<var name="bergHeight"/>
-			<var name="uBergVelocity"/>
-			<var name="vBergVelocity"/>
 			<var name="testArrayRegression"/>
 			<var name="testArrayParallelism"/>
 			<var name="testArrayRestartability"/>
 			<var name="forcingGroupNames"/>
 			<var name="forcingGroupRestartTimes"/>
+			<var name="bergAreaCategory"/>
+			<var name="bergLength"/>
+			<var name="bergHeight"/>
+			<var name="bergAreaCell"/>
+			<var name="bergMassCategory"/>
+			<var name="bergMassCell"/>
+			<var name="uBergVelocity"/>
+			<var name="vBergVelocity"/>
+			<var name="bergVelocityTimeStep"/>
+			<var name="bergAdvectionTimeStep"/>
```

Comment:
These timesteps vary??

Reply:
They can, if advection is done outside the velocity solver subcycling.

Reply:
They vary from sea ice time step to time step? Like they're adaptive?

Reply:
Ah no, not adaptive. They're set in seaice_init_berg_velocity_solver, which is only called if the run is an initial run, which is why I've put them in the restart file. Let me know if you'd prefer it structured a different way.

Reply:
I guess this approach doesn't allow for changing the timestep size / number of subcycling steps in a restart file, which isn't desirable. I'll change this so the timesteps are always computed at the start of a run, whether initial or restart.

Reply:
Yes, thats whats done in the sea ice velocity solver. Since it can be computed at the start it should, if only to avoid cluttering the restart file with things that dont need to be there